packages/bpk-component-popover/src/BpkPopoverPortal.js

```diff
@@ -18,92 +18,67 @@
 
 /* @flow */
 
+import Popper from 'popper.js';
+import PropTypes from 'prop-types';
+import React, { Component, type Node } from 'react';
 import focusStore from 'a11y-focus-store';
 import focusScope from 'a11y-focus-scope';
 import { Portal, cssModules } from 'bpk-react-utils';
-import PropTypes from 'prop-types';
-import React, { Component, type Node } from 'react';
-import Tether, {
-  getArrowPositionCallback,
-  applyRTLTransforms,
-} from 'bpk-tether';
 
 import STYLES from './bpk-popover.scss';
 import BpkPopover, { type Props as PopoverProps } from './BpkPopover';
 import { ARROW_ID } from './constants';
 
 const getClassName = cssModules(STYLES);
 
-const onOpen = (popoverElement, targetElement) => {
-  // If the target element does _not_ exist, the Portal does not open
-  // Therefore we also shouldn't store and scope the focus
-  if (targetElement) {
-    targetElement.focus();
-    focusStore.storeFocus();
-    focusScope.scopeFocus(popoverElement);
-  }
-};
-
 export type Props = {
   ...$Exact<PopoverProps>,
   target: (() => HTMLElement) | Node,
   isOpen: boolean,
-  tetherOptions: Object,
+  placement: ?('top' | 'right' | 'bottom' | 'left'),
   portalStyle: ?Object,
   portalClassName: ?string,
   renderTarget: ?() => HTMLElement,
 };
 
 class BpkPopoverPortal extends Component<Props> {
-  tether: ?Tether;
+  popper: ?Popper;
+  previousTargetElement: ?HTMLElement;
 
   static propTypes = {
     ...BpkPopover.propTypes,
     target: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
     isOpen: PropTypes.bool.isRequired,
     onClose: PropTypes.func.isRequired,
-    tetherOptions: PropTypes.shape({
-      attachment: PropTypes.string,
-      targetAttachment: PropTypes.string,
-      offset: PropTypes.string,
-      constraints: PropTypes.array,
-    }),
```

Comment:
Is all this just redundant / implicit now as part of Popper?

Reply:
So i've added `placement` as a prop now - this can be used to position the popover instead of `tetherOptions`. Its reduced functionality, but folk don't tend to customise it too much.